.ci.yaml

```diff
@@ -210,6 +210,15 @@ targets:
     timeout: 90
     scheduler: luci
 
+  - name: Linux linux_host_engine
+    recipe: engine_v2/engine_v2
+    bringup: true
+    timeout: 60
+    properties:
+      config_name: linux_host_engine
+      environment: Staging
```

Comment:
is environment necessary? The tasks have bringup passed to them

Reply:
Yes, we'll need to identify staging, prod, try, etc. This property is used to manage the destination of the archives and select the correct drone builders.

Reply:
From go/flutter-staging-infra, we could generate this property (but it's fine to leave in this PR)

Reply:
I haven't validated these builds from the other environments. It will most likely require more changes in the recipes.